TESTS/netsocket/ethernet.json

```diff
@@ -0,0 +1,27 @@
+{
+    "config": {
+        "header-file": {
+            "help" : "String for including your driver header file",
+            "value" : "\"EthernetInterface.h\""
+        },
+        "object-construction" : {
```

Comment:
It is so that different network interfaces can be tested without modifying the code base. IE, if someone wanted to test ESP8266, they would add the driver for it, update this file, then run the tests.